gli4py/glinet.py

```diff
@@ -171,6 +170,23 @@ async def connected_clients(self) -> dict:
                 clients[client['mac']] = client
         return clients
 
+    async def _wifi_config_get(self) -> dict:
+        return await self._request(self.gen_sid_payload('call', ['wifi', 'get_config'], self.sid))
+
+    async def _wifi_config_set(self, config: dict) -> dict:
+        return await self._request(self.gen_sid_payload('call', ['wifi', 'set_config', config], self.sid))
+    
+    async def wifi_ifaces_get(self) -> dict:
+        wifi_config = await self._wifi_config_get()
+        return {iface.get('name'):iface for dev in wifi_config.get('res', []) for iface in dev.get('ifaces')}

```

Comment:
This is not very readable, please do over several lines and add some typing if applicable.